µ.go

```diff
@@ -14,17 +14,43 @@
 // limitations under the License.
 //
 
+// mugo transpiles a subset of the Go language to C++.
+//
+// See package transpiler for more details.
 package main
 
 import (
+	"errors"
+	"flag"
+	"fmt"
+	"go/ast"
+	"io/ioutil"
 	"log"
 	"os"
 
 	"github.com/googlesamples/mugo/transpiler"
 )
 
+func mainImpl() error {
```

Comment:
I think this is small enough it could get merged with `main`, what do you think?